lib/new-admin/admin-server.js

```diff
@@ -33,23 +37,57 @@ app.use(helmet())
 app.use(compression())
 app.use(nocache())
 app.use(cookieParser())
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true })) // support encoded bodies
 app.use(express.static(path.resolve(__dirname, '..', '..', 'public')))
 
+app.use(['*'], session({
+  store: new pgSession({
+    pgPromise: db,
+    tableName: 'user_sessions'
+  }),
+  name: 'lid',
+  secret: 'MY_SECRET',
```

Comment:
Should the secret be hard coded in the source code? I was thinking of an .env variable

Reply:
You are correct, this should be an env var. I'll raise this question again with Rafael and fix this